docs/commands.rst

```diff
@@ -245,7 +245,7 @@ The command ``inductive`` can be used to define an inductive type, its construct
    ￼set builtin "Prop" ≔ ... // : TYPE
    ￼set builtin "P"    ≔ ... // : Prop → TYPE
 ￼
-For the moment, we only support first-order data types, with dependent and polymorphic types.
+For the moment, we only support first-order data types, potentially mutually dependent, with dependent and polymorphic types.
```

Comment:
For the moment, we only support (mutually defined) first-order dependent types. Polymorphic types can be encoded by defining a type Set and a function τ:Set→TYPE.